_blog/2021-12-29-2021 Recap.md

```diff
@@ -0,0 +1,37 @@
+---
+title: 2021 Recap
+excerpt: What happened in the AntennaPod project in 2021
+date: "2021-12-29 11:00:00"
+image: "2021/recap.png"
+author: bytehamster
+layout: blog
+---
+
+Looking back at 2021, a lot happened in the AntennaPod community. In this blog post, we want to summarize the most important milestones.
+
+# App development
+**Over the course of 2021**, 48 contributors helped improve AntennaPod. In total, they made no less than 482 changes (commits) to the source code, touching about 29 000 lines of code. This makes AntennaPod a pretty active project, even though it is run completely by volunteers in their free-time. To give everyone access to the app improvements, we released 9 new updates, of which 3 contained major new features.
```

Comment:
```suggestion
**Over the course of 2021**, 48 contributors helped improve AntennaPod. In total, they made no less than 482 changes (commits) to the source code, touching about 29 000 lines of code. This makes AntennaPod a pretty active project, even though it is run completely by volunteers in their free time.

To give everyone access to the app improvements, we released 9 new updates, of which 3 contained major new features.
```
(There currently is no white line, and in retrospect this last sentence better fits with the next bit.)